task-service/openapi_tasklist.yaml

```diff
@@ -0,0 +1,146 @@
+openapi: 3.0.3
+info:
+  title: Task List API
+  description: API for managing tasks list
+  version: 0.0.1
+servers:
+  - url: '{host}/api'
+    variables:
+      host:
+        default: http://localhost:8080
+paths:
+  /tasks/v1:
```

Comment:
can a task be outside a directory? If not, I would rather follow a proper path
`/directories/{directoryId}/tasks/taskId`